### PR TITLE
README changes

### PR DESCRIPTION
**Pull request by sahil-lakhwani**
Signed-off-by: sahil-lakhwani <sahilakhwani@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,79 @@
-# kbrew
```
Comment (PrasadG193) [inline README.md:1]: I think we should keep the title here

File: README.md [hunk 2]
```diff
@@ -1,26 +1,79 @@
...
+
+With kbrew, all of this happens with a "one step":
 
 ```
 $ kbrew install kafka-operator
 ```
-## Helm chart or operator or Manifests - all abstracted
 
-kbrew abstracts the underlying chart or operator or manifest and gives you a recipe to install a stack with all basic configurations done.
+Similarly when you install a Rook Ceph operator - it makes it a `one step easy`:
```
Comment (PrasadG193) [inline README.md:25]:
```suggestion
Similarly when you install a Rook Ceph cluster - it makes it a `one step easy`:
```

File: README.md [hunk 3]
```diff
@@ -1,26 +1,79 @@
...
+   * [Recipe structure](#recipe-structure)
+      * [Application](#application)
+         * [Arguments](#arguments)
+      * [Pre &amp; Post Install](#pre--post-install)
+      * [Pre and Post Cleanup](#pre-and-post-cleanup)
+* [FAQ](#faq)
+         * [Should I use kbrew for installing applications in production environment?](#should-i-use-kbrew-for-installing-applications-in-production-environment)
+         * [How can I contribute recipes for a project/tool?](#how-can-i-contribute-recipes-for-a-projecttool)
+         * [How is analytics used?](#how-is-analytics-used)
+         * [Who is developing kbrew?](#who-is-developing-kbrew)
```
Comment (PrasadG193) [inline README.md:60]: This is not getting rendered correctly 
```suggestion
* [FAQ](#faq)
  * [Should I use kbrew for installing applications in production environment?](#should-i-use-kbrew-for-installing-applications-in-production-environment)
  * [How can I contribute recipes for a project/tool?](#how-can-i-contribute-recipes-for-a-projecttool)
  * [How is analytics used?](#how-is-analytics-used)
  * [Who is developing kbrew?](#who-is-developing-kbrew)
```

Comment (vishal-biyani) [reply README.md]: Never mind, worked now!

**Pull request merged**